pkg/mcs/scheduling/server/cluster.go

```diff
@@ -348,3 +348,28 @@ func (c *Cluster) HandleStoreHeartbeat(heartbeat *schedulingpb.StoreHeartbeatReq
 	c.hotStat.CheckReadAsync(statistics.NewCollectUnReportedPeerTask(storeID, regions, interval))
 	return nil
 }
+
+// RunUpdateStoreStats updates store stats periodically.
+func (c *Cluster) RunUpdateStoreStats() {
+	defer logutil.LogPanic()
+
+	ticker := time.NewTicker(9 * time.Millisecond)
```

Comment:
9 * time.Millisecond?

Reply:
keep the same as previous and less than the interval of store heartbeat.

Reply:
Keep the same as previous.